sklearn/metrics/tests/test_common.py

```diff
@@ -972,7 +1016,7 @@ def check_sample_weight_invariance(name, metric, y1, y2):
         weighted_score, weighted_score_list,
         err_msg=("Weighted scores for array and list "
                  "sample_weight input are not equal (%f != %f) for %s") % (
-                     weighted_score, weighted_score_list, name))
+                    weighted_score, weighted_score_list, name))
```

Comment:
Why did you change indentation?

Reply:
This wasn't intentional, actually in between commits I'd to touch this section of code, but later on I reverted the changes made by those commits, leaving behind this trace of "non-wanted" indentation.